docs/source/quickstart.mdx

```diff
@@ -0,0 +1,545 @@
+  {
+    {
+      "data": {
+        "listAllProducts": [
+          {
+            "id": "product:1",
+            "title": "Air Jordan 1 Mid",
+            "description": "Air Jordan 1 Mid is a blue, grey and white sneaker from the iconic jordan brand",
+            "mediaUrl": "https://sneakernews.com/wp-content/uploads/2022/06/air-jordan-1-mid-university-blue-grey-dx9276-100-6.jpg",
+            "variants": [
+              {
+                "id": "variant:1",
+                "price": 600.25
+              },
+              {
+                "id": "variant:2",
+                "price": 20.12
+              }
+            ]
+          },
+          {
+            "id": "product:2",
+            "title": "Supreme x Tiffany & Co. Box Logo Tee",
+            "description": "A classic Supreme vbox t-shirt in the signature Tiffany blue.",
+            "mediaUrl": "https://encrypted-tbn2.gstatic.com/shopping?q=tbn:ANd9GcQWDHD3SSS98UAVKODaql7nrDTopfL4tcTnEltW8Yqy4hyDu4i5b70Wb3Y8-wACJIo5g-ZdRULPQKUmt7JfwiaSdgiOBz4pvU_YelKHUI4nhoXmMJPeh_tyWQ",
+            "variants": [
+              {
+                "id": "variant:4",
+                "price": 600.25
+              },
+              {
+                "id": "variant:5",
+                "price": 600.25
+              },
+              {
+                "id": "variant:6",
+                "price": 600.25
+              },
+              {
+                "id": "variant:7",
+                "price": 600.25
+              },
+              {
+                "id": "variant:8",
+                "price": 600.25
+              }
+            ]
+          },
+          {
+            "id": "product:3",
+            "title": "THE MACKINAC 40MM",
+            "description": "Established by Detroit’s historic Bayview Yacht club, the days-long Port Huron to Mackinac Island regatta is one of the longest and most grueling freshwater races in the world.\n\nNamed for this legendary competition, the Shinola Mackinac is our first watch with automatic, single-eye chronograph yacht-timer functionality.\n\nIt’s a precision instrument designed to be passed on for generations—just like the tradition that inspires it.",
+            "mediaUrl": "https://shinola-m2.imgix.net/images/Products/20253783-sdt-012455107/S0120253783_F2_MAIN_01.png?h=1500&w=1500&bg=f7f7f7&auto=format,compress&fit=fillmax",
+            "variants": [
+              {
+                "id": "variant:9",
+                "price": 3499.99
+              }
+            ]
+          },
+        ]
+      }
+    },
+    ...
+  }
+  ```
+
+  </ExpansionPanel>
+
+  This operation requests a list of all products from the `products` service.
+  You can remove fields like `mediaUrl` or add fields like `releaseDate` to the query to see how the reponse changes.
+
+  Next, you'll execute a query that demonstrates the power of federation.
+
+4. Replace the example query in the **Operation** window with the following one, but don't run it yet.
```

Comment:
"but don't run it yet" asking them to do something while preventing them from executing is a bit anticlimactic or cliffhangeresque (I invent words).

suggestion: letting them execute, have an error message, and explaining what it needs to work?

Reply:
Was thinking about this as well! In retrospect, it's a better way of doing it.

Reply:
https://github.com/apollographql/federation/pull/3037/commits/5fc4f6e6c127cc23544f1452ab5c0381b6c865da